rolling/distribution.yaml

```diff
@@ -4463,5 +4463,21 @@ repositories:
       url: https://github.com/ros2/yaml_cpp_vendor.git
       version: master
     status: maintained
+  zenoh-bridge-dds:
```

Comment:
Small problem, [REP 144](https://ros.org/reps/rep-0144.html) only allows package names to use underscores `_`, not dashes `-`. Would you be willing to rename the package inside the repository? I don't think the repository name on github needs to change, but you may need to use underscores too when running bloom after changing the name in the `CMakeLists.txt` and `package.xml`.

`bloom-release --rosdistro rolling --track rolling zenoh_bridge_dds`

Reply:
OK, I see.
I can't rename the source repo. But for coherency, I'll rename the release repo according to the package name: `zenoh_bridge_dds-release`. Then I'll `bloom-release` again which will probably create another PR. Thus I'll close this one.

That's unfortunate the `ros2 pkg create` command doesn't print at least some warning for this 😕